importScripts/sampleRecords/library.destination.json

```diff
@@ -4,7 +4,6 @@
     {
       "object_id":"1",
       "id":"1",
-      "idno":"10",
```

Comment:
`idno` values were cleaned by the client. Previously we were loading PublicationID in as the default `idno` of no `Accession Number` has been provided but this is inaccurate. This PR now removes these values.